src/cogent3/__init__.py

```diff
@@ -405,9 +422,11 @@ def load_unaligned_seqs(
         return load_from_json(filename, (SequenceCollection,))
 
     data = _load_seqs(file_format, filename, format, kw, parser_kw)
-    if new_type and moltype is None:
-        raise ValueError("Argument 'moltype' is required when 'new_type=True'")
-    elif new_type:
+
+    if new_type or "COGENT3_NEW_TYPE" in os.environ:
```

Comment:
no need to have these checks within the `load_...` functions since their counterpart `make_...` function does the checking. Delete all such redundancies